framework/src/main/java/org/checkerframework/common/wholeprograminference/SceneToStubWriter.java

```diff
@@ -450,6 +452,11 @@ private static boolean isInternalJDKAnnotation(String annotationName) {
     private static int printClassDefinitions(
             String basename, AClass aClass, PrintWriter printWriter) {
         String[] classNames = basename.split("\\$");
+        TypeElement innermostTypeElt = aClass.getTypeElement();
+        TypeElement[] typeElements = getTypeElementsForClasses(innermostTypeElt, classNames);
+        if (innermostTypeElt == null) {
```

Comment:
I would test `innermostTypeElt` before using it; that is, transpose the two statements.